django_loose_fk/loaders.py

```diff
@@ -54,6 +55,10 @@ def is_local_url(self, url: str) -> bool:
                 "break django-loose-fk's behaviour. You should use an explicit list.",
                 RuntimeWarning,
             )
+            return validate_host(parsed.netloc, allowed_hosts)
```

Comment:
I was wondering if there is a particular reason we don't do `validate_host` in the `is_local` util itself? I see that in `filters.py` and `drf.py` `is_local` is used without validating if it matches with `ALLOWED_HOSTS`

Reply:
It was not developed by me, but I have a theory:) in the Resolver we have `host` as parameter and it's used where we can use `host` from request. Here we don't know `host`, therefore we compare it with ALLOWED_HOSTS

Reply:
Ah that makes sense :+1: